### ISSUE TITLE
TypeError on Fedora

### ISSUE THREAD

**Issue by kallisti5** [pipiliang/docker-dashboard#15]
```
$ sudo docker-dashboard
TypeError: Cannot read property 'IPAddress' of undefined
    at Object.utils.toRow (/usr/lib/node_modules/docker-dashboard/lib/utils.js:17:79)
    at containers.forEach.containerInfo (/usr/lib/node_modules/docker-dashboard/lib/widget-container.js:75:20)
    at Array.forEach (<anonymous>)
    at Object.callback (/usr/lib/node_modules/docker-dashboard/lib/widget-container.js:74:14)
    at /usr/lib/node_modules/docker-dashboard/node_modules/dockerode/lib/docker.js:429:12
    at Modem.buildPayload (/usr/lib/node_modules/docker-dashboard/node_modules/docker-modem/lib/modem.js:270:7)
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/docker-dashboard/node_modules/docker-modem/lib/modem.js:229:14)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
```

```
$ sudo docker-dashboard
TypeError: Cannot read property '0' of null
    at images.forEach.s (/usr/lib/node_modules/docker-dashboard/lib/widget-images.js:26:23)
    at Array.forEach (<anonymous>)
    at Object.self.docker.listImages [as callback] (/usr/lib/node_modules/docker-dashboard/lib/widget-images.js:23:10)
    at /usr/lib/node_modules/docker-dashboard/node_modules/dockerode/lib/docker.js:465:12
    at Modem.buildPayload (/usr/lib/node_modules/docker-dashboard/node_modules/docker-modem/lib/modem.js:270:7)
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/docker-dashboard/node_modules/docker-modem/lib/modem.js:229:14)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
```

**Comment by kallisti5** [comment]
```
$ docker -v
Docker version 1.13.1, build bdb8293-unsupported
```

**Comment by pipiliang** [comment]
This issue was fixed in v0.1.5，try again after the update。
